Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml

```
@@ -78,6 +78,10 @@
     - state: margherita
   - type: SliceableFood
     slice: FoodPizzaMargheritaSlice
+  - type: Tag
+    tags:
+    - Pizza
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)